### PR TITLE
Fix certain item drops

### PR DESCRIPTION
**Pull request by mathiascode**
Meta shouldn't be reset for these blocks, as they have special item forms.

### CODE REVIEW

File: src/Blocks/BlockComparator.h [hunk 1]
```diff
@@ -3,16 +3,17 @@
 
 #include "BlockHandler.h"
 #include "BlockRedstoneRepeater.h"
+#include "BlockStairs.h"
```
Comment (bearbin) [inline src/Blocks/BlockComparator.h]: What is this include needed for?

Comment (mathiascode) [reply src/Blocks/BlockComparator.h]: Leftover, will remove.

File: src/Blocks/BlockRedstoneRepeater.h [hunk 2]
```diff
@@ -5,16 +5,17 @@
 #include "Mixins.h"
 #include "ChunkInterface.h"
 #include "BlockSlab.h"
+#include "BlockStairs.h"
```
Comment (bearbin) [inline src/Blocks/BlockRedstoneRepeater.h]: There's another one here.

**Comment by bearbin** [comment]
This looks good once style and above comment is fixed.

**Pull request merged**